main.js

```diff
@@ -116,6 +116,19 @@ let BiggerBark = new class extends Upgrade { // A bigger bark.
   }
 }
 
+new class extends Upgrade { // All upgrades extend Upgrade.
+    meta () { // Info about your upgrade.
+      return {
+    "name": "make bruh beagle fat",
+    "desc": "working 2012",
```

Comment:
please make it work in 2013 :trollface:

Reply:
```suggestion
    "desc": "working 2013",
```

i think this would work

Reply:
then it wouldn't work in 2012

Reply:
i fixed it!!
#12 and #13